packages/webdriverjs/test/axe-webdriverjs.spec.ts

```diff
@@ -619,7 +616,7 @@ describe('@axe-core/webdriverjs', () => {
     });
 
     it('with chaining include and exclude', async () => {
-      await driver.get(`${addr}/context.html`);
+      await driver.get(`${addr}/context-include-exclude.html`);
```

Comment:
This test doesn't really prove anything. If you only had the .includes the assertions would still pass because the excludes aren't nested in an include. Not sure that needs to be fixed in this PR, but something should be done about it IMO.

Reply:
Yup. Noticed that too. This would fix it https://github.com/dequelabs/axe-test-fixtures/pull/28